ArduCopter/Copter.h

```diff
@@ -725,7 +736,25 @@ class Copter : public AP_Vehicle {
     void set_accel_throttle_I_from_pilot_throttle();
     void rotate_body_frame_to_NE(float &x, float &y);
     uint16_t get_pilot_speed_dn() const;
-    void run_rate_controller();
+    void run_rate_controller_main();
+
+    // if AP_INERTIALSENSOR_FAST_SAMPLE_WINDOW_ENABLED
```

Comment:
Yes, because of the vehicle include protections it can't actually be defined here, the comment is just to indicate that the code is technically controlled